vignettes/algorithm.Rmd

```diff
@@ -157,7 +157,7 @@ For example, for an unstructured covariance matrix, $\theta$ has $k = G * m(m+1)
 
 #### Spatial covariance matrix
 
-Under spatial covariance structure, the covariance matrix is different for all subjects, if they have different visit timepoints.
+Under spatial covariance structure, the covariance matrix is different for all subjects, if they have different time points.
```

Comment:
suggestion:
"A spatial covariance structure can model individual-specific visit times. An individuals covariance matrix is then a function of both the population-level covariance parameters (specific to the chosen structure) and the individuals visit times."

ps: I know sas calls those spatial covariance structures but this just shows how confusing that terminology is x)

Reply:
done :-)